core/object/script_language.cpp

```diff
@@ -708,7 +708,12 @@ void PlaceHolderScriptInstance::property_set_fallback(const StringName &p_name,
 			}
 		}
 		if (!found) {
-			properties.push_back(PropertyInfo(p_value.get_type(), p_name, PROPERTY_HINT_NONE, "", PROPERTY_USAGE_NO_EDITOR | PROPERTY_USAGE_SCRIPT_VARIABLE));
+			const Variant::Type type = p_value.get_type();
+			PropertyHint hint = PROPERTY_HINT_NONE;
+			if (type == Variant::OBJECT && ((Object *)p_value)->is_class("Node")) {
```

Comment:
This isn't guaranteed to be safe I'd say, it could be `nullptr` or freed, so I'd use:
```suggestion
			Object *obj = p_value.get_validated_object();
			if (obj && obj->is_class("Node")) {
```

Reply:
Thanks for always noticing my mistakes 🙈